### DISCUSSION TITLE
What is all needed to run this locally for Polkdot?

### DISCUSSION THREAD

**Discussion by timmolter** [itering/subscan-essentials#173] [Q&A] [answered]
If I understand correctly, this project will enable a REST API described here: https://docs.api.subscan.io/#introduction

But what else is needed? I don't see any configuration for a Polkadot node (https://github.com/paritytech/polkadot/tags). Is project just communicating with a MySQL database? Or directly to a node?

**Comment by freehere107** [comment, marked as answer]
@timmolter https://github.com/subscan-explorer/subscan-essentials/blob/master/util/env.go#L13 Regarding the network node, you need to set a websocket endpoint here
